tests/spec/puppeteer-spec.js

```diff
@@ -0,0 +1,98 @@
+jest.setTimeout(20000);
+
+describe("An editor for MicroPython running at localhost.", function() {
+
+    beforeAll(async() => {
+        global.browser = await global.puppeteer.launch({headless: true, args: ['--no-sandbox', '--disable-setuid-sandbox', '--disable-dev-shm-usage']});
+    });
+
+    afterAll(async() => {
+        global.browser.close();
+    });
+
+    it("Correctly loads a script from an import URL.", async function() {
+
+        const page = await global.browser.newPage();
+        const projectURL = "http://localhost:5000/editor.html?#project:XQAAgAApAQAAAAAAAAA9gn0iDP5hOXUMBZ4M1sxt7nhTa/UMRecCSq6uHLM44uVEs1hTA1G/Oa3Hy9fjvqw9MOrrvyqKstR9g9oq4yc4pkk1m9E2hvucCCCVEeUdb6bwT0S5asuGStzirbKaXcmYjTAskliKk/1v60vUCxCI/fc8ZUstwqzchTG2zAzzDii/EzhUsce8bjtDMg+OOMAzY03WeyEN6x5Z3bkVA20HbuSfofyGzVIlKfTxKeZlZVU2Wt3DdOqe1ccGelN7y0dADIpV19vKoZ9AWI8K4l3FkQQ43EIIM/vyyq0+JjpgrLhtSv/8Ma+A";
+        await page.goto(projectURL);
+
+        let hasLoadedContent = false;
+        const codeContent = await page.evaluate("window.EDITOR.getCode();");
+        if (codeContent.includes("Test passed!") && codeContent.length === 111) hasLoadedContent = true;
```

Comment:
If these are the condition we are checking for the test to pass, then can we move those to the "assert" section to AAA (Arrange-Act-Assert)?

```js
const codeContent = await page.evaluate("window.EDITOR.getCode();");
await page.close();

expect(codeContent).toHaveLength(111);
expect(codeContent).toContain("Test passed!");
```

Reply:
LGTM.

Reply:
Changed in https://github.com/microbit-rosslowe/PythonEditor/pull/9/commits/3dcdaf967f1b0b1ec45374b6e936c721b0e0781d